edge-modules/TestAnalyzer/PartitionReceiverHandler.cs

```diff
@@ -77,20 +75,20 @@ static DateTime GetEnqueuedTime(string deviceId, string moduleId, EventData even
                 }
                 else
                 {
-                    Log.LogError($"Message for module [{moduleId}] and device [{deviceId}] enqueued time [{enqueued}] cannot be parsed.");
+                    Logger.LogError($"Message for module [{moduleId}] and device [{deviceId}] enqueued time [{enqueued}] cannot be parsed.");
                 }
             }
             else
             {
-                Log.LogError($"Message for module [{moduleId}] and device [{deviceId}] doesn't contain {EnqueuedTimePropertyName} property.");
+                Logger.LogError($"Message for module [{moduleId}] and device [{deviceId}] doesn't contain {EnqueuedTimePropertyName} property.");
             }
 
             return enqueuedtime;
         }
 
         public Task ProcessErrorAsync(Exception error)
```

Comment:
no usage; please remove it.

Reply:
What if the TryGetValue() fails? If eventhub changes how they tag messages, we should not eat this failure.